uamqp/client.py

```diff
@@ -555,9 +562,6 @@ def _message_received(self, message):
             self._message_received_callback(message)
         if not self._streaming_receive:
             self._received_messages.put(message)
-        elif not message.settled:
```

Comment:
how about comment out those lines, and add a TODO: do we need settled property for a message?